### PR TITLE
Features/graylog

### PR DESCRIPTION
**Pull request by duythinht**
I just add support for graylogs. Tested working with graylog2 version >= 1.3.4
@lonelycode Please take a reviews

### CODE REVIEW

File: pumps/graylog.go [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+		if err != nil {
+			log.WithFields(logrus.Fields{
+				"prefix": graylogPrefix,
+			}).Fatal(err)
+		}
+
+		gelfData := map[string]interface{}{
+			//"version": "1.1",
+			"host":      "tyk-pumps",
+			"timestamp": record.TimeStamp.Unix(),
```
Comment (richard-hulm) [inline pumps/graylog.go]: Do you not want to use the timestamp from the analytics data rather than the current timestamp?

Comment (duythinht) [inline pumps/graylog.go]: It was using analytics timestamp from the record

Comment (richard-hulm) [inline pumps/graylog.go]: .......Completely misread that sorry!

**Comment by lonelycode** [comment]
LGTM :-)

**Pull request merged**